examples/use_notifications.py

```
@@ -0,0 +1,275 @@
+    print_notification(notification)
+
+    # Add notifications of different types
+    print("\n3. Adding notifications of different types")
+
+    info_id = add_notification(
+        "This is an informational message",
+        notification_type=NotificationType.INFO,
+        async_delivery=False,
+    )
+
+    success_id = add_notification(
```

Ruff (0.8.2)

Local variable `success_id` is assigned to but never used

Remove assignment to unused variable `success_id`

(F841)